lua/github-theme/theme.lua

```diff
@@ -453,6 +453,25 @@ theme.setup = function(cfg)
     ALEWarningSign = { fg = c.warning },
     ALEErrorSign = { fg = c.error },
 
+    -- Barbar
+    BufferCurrent = { fg = c.fg },
+    BufferCurrentIndex = { bg = c.bg, fg = c.fg },
+    BufferCurrentMod = { bg = c.bg, fg = c.warning },
+    BufferCurrentSign = { bg = c.bg, fg = c.info },
+    BufferCurrentTarget = { bg = c.bg, fg = c.fg_light },
+    BufferVisible = { bg = c.bg2, fg = c.fg_dark },
+    BufferVisibleIndex = { bg = c.bg2, fg = c.red },
+    BufferVisibleMod = { bg = c.bg2, fg = c.warning },
+    BufferVisibleSign = { bg = c.bg2, fg = c.info },
+    BufferVisibleTarget = { bg = c.bg2, fg = c.red },
+    BufferInactive = { bg = c.bg2, fg = c.fg_dark },
+    BufferInactiveIndex = { bg = c.bg2, fg = c.fg_light },
+    BufferInactiveMod = { bg = c.bg2, fg = util.darken(c.warning, 0.7) },
+    BufferInactiveSign = { bg = c.bg2, fg = util.darken(c.fg_dark, 0.4, c.bg2) },
+    BufferInactiveTarget = { bg = c.bg2, fg = c.red },
+    BufferTabpages = { bg = c.bg2, fg = c.none },
+    BufferTabpageFill = { bg = c.bg2, fg = c.bg_highlight },
```

Comment:
ℹ️ This was c.border_highlight in the original PR but this seems to be the closest equivalent. It looks great to me.